scenario/fixtures/chaincode/golang/basic/main.go

```diff
@@ -77,3 +109,12 @@ func toStringValues(input map[string][]byte) map[string]string {
 
 	return results
 }
+
+func contains(slice []string, entry string) bool {
+	for _, item := range slice {
+		if entry == item {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
I don't see this used anywhere. Could it be omitted?

Reply:
Yes, it got left in there from an earlier experiment